src/org/zaproxy/zap/extension/brk/ExtensionBreak.java

```diff
@@ -153,17 +168,14 @@ public void hook(ExtensionHook extensionHook) {
 	        extensionHook.getHookMenu().addToolsMenuItem(getMenuDrop());
 	        extensionHook.getHookMenu().addToolsMenuItem(getMenuAddHttpBreakpoint());
             
-            extensionHook.addProxyListener(getProxyListenerBreak());
-
-            // APIs are usually loaded even if the view is null, as they are specifically for daemon mode
-            // However in this case the API isnt really of any use unless the UI is available
-            extensionHook.addApiImplementor(api);
-
-            extensionHook.addSessionListener(this);
-            extensionHook.addOptionsChangedListener(this);
-
-	    	ExtensionHelp.enableHelpKey(getBreakPanel(), "ui.tabs.break");
+	    	ExtensionHelp.enableHelpKey(breakPanel, "ui.tabs.break");
 	    	ExtensionHelp.enableHelpKey(getBreakpointsPanel(), "ui.tabs.breakpoints");
+	    } else {
+	    	this.breakpointManagementInterface = 
+	    			new HttpBreakpointManagementDaemonImpl(getProxyListenerBreak());
+
+	    	breakpointMessageHandler = new BreakpointMessageHandler2(breakpointManagementInterface);
+	    	breakpointMessageHandler.setEnabledBreakpoints(new ArrayList<BreakpointMessageInterface>());

```

Comment:
think I've addressed all of those points. Some of the indentation issues are due to the mixed tab and space chrs already used in that file - I didnt want to change to make them all consistent in this PR :/